types/sharedb/lib/sharedb.d.ts

```diff
@@ -143,9 +143,9 @@ export interface DocEventMap<T> {
     'no write pending': () => void;
     'nothing pending': () => void;
     'create': (source: any) => void;
-    'op': (ops: [any], source: any) => void;
+    'op': (ops: [any], source: any, clientId: any) => void;
```

Comment:
```suggestion
    'op': (ops: [any], source: any, clientId: string) => void;
```

We should match the (upcoming) update to [`Connection`](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54821/files#diff-fd86d2eab56961fbfa694b2ab8f4f1ae526007789b15f1ac4e4ffcc9c06ec459R17)